peekingduck/pipeline/nodes/output/csv_writer.py

```diff
@@ -24,8 +24,29 @@
 from peekingduck.pipeline.nodes.node import AbstractNode
 from peekingduck.pipeline.nodes.output.utils.csvlogger import CSVLogger
 
+
 class Node(AbstractNode):
-    """Node that output a csv with user input choice of statistics to track"""
+    """Node that output a csv with user input choice of statistics to track.
+
+    Inputs:
+        All
```

Comment:
since All is unique here, can add a short description for people to refer to `stats_to_track` to select items to write

Reply:
Added.

Reply:
hmm. move it down under `inputs` and see if you can adhere to the same format as the other nodes. If not, `All` will just look like a very normal text in RTD